.eslintrc.json

```diff
@@ -11,9 +11,8 @@
     "eslint:recommended",
     "plugin:react/recommended",
     "plugin:jsx-a11y/recommended",
-    "plugin:react-hooks/recommended",
-    "plugin:jest/recommended",
-    "plugin:testing-library/react"
+    "plugin:react-hooks/recommended"
+    // "plugin:testing-library/react"
```

Comment:
I assume we removed `jest` from VS Code because it was running tests with every change right?

We actually turn off auto run by adding the key-value pair to our `settings.json` in VS Code:

`"jest.autoRun": "off"`